api/views.py

```diff
@@ -17,7 +19,13 @@ def get(self, request, format=None):
             'export-badges': reverse('api:export-badges', request=request,
                                      format=format),
             'export-instructors': reverse('api:export-instructors',
-                                          request=request, format=format)
+                                          request=request, format=format),
+            'events-past': reverse('api:events-past',
+                                   request=request, format=format),
+            'events-ongoing': reverse('api:events-ongoing',
+                                      request=request, format=format),
+            'events-upcoming': reverse('api:events-upcoming',
+                                       request=request, format=format),
```

Comment:
And will you add a fourth which is "all published events, unfiltered"?  I think that's what we want for building the website.